File: CC/Codigos_Fonte_Relatorio_1/micro09/micro09.java

public class micro09{

    public static void main(String[] args){
        double preco, venda, novo_preco;

        System.out.println("Digite o preço: ");
        preco = Double.parseDouble(System.console().readLine());

        System.out.println("Digite a venda: ");
        venda = Double.parseDouble(System.console().readLine());

        if((venda < 500) || (preco < 30)) novo_preco = preco + ((10/100) * preco);
        else{

            if(((venda >= 500) && (venda < 1200)) || ((preco >= 30) && (preco < 80))) novo_preco = preco - ((15/100) * preco);
            else if((venda >= 1200) || (preco >= 80)) novo_preco = preco - ((20/100)*preco);

        }         
        
    }
}

File: CC/Codigos_Fonte_Relatorio_1/micro09/micro09.pas
program micro09;

var
    preco, venda, novo_preco : real;

begin
    write('Digite o preço: ');
    read(preco);

    write('Digite a venda: ');
    read(venda);

    if((venda < 500) or (preco < 30)) then 
        novo_preco := preco + ((10/100) * preco)
    
    else
    begin
        if(((venda >= 500) and (venda < 1200)) or ((preco >= 30) and (preco < 80))) then 
            novo_preco := preco - ((15/100) * preco)
            
        else if((venda >= 1200) or (preco >= 80)) then
            novo_preco := preco - ((20/100)*preco);
    end;

end.
